xcell/mappers/mapper_LenzCIB.py

```diff
@@ -11,4 +11,6 @@ def __init__(self, config):
          'nside':512}
         """
         self._get_Planck_defaults(config)
-        self.beam_info = config.get('beam_fwhm_arcmin', 5.)
+        self.beam_info = config.get('beam_fwhm_arcmin',
+                                    {'type': 'Gaussian',
+                                     'FWHM_arcmin': 5.0})
```

Comment:
I think this section of the yaml file should be called `'beam_info'` in general, not `'beam_fwhm_arcmin'`

Reply:
yes, that was the intention. This is a brain fart.